### PR TITLE
适配AirGo面板

### PR DESCRIPTION
**Pull request by ppoonk**
适配AirGo面板

### CODE REVIEW

File: release/config/config.yml.example [hunk 1]
```diff
@@ -13,7 +13,7 @@ ConnectionConfig:
   DownlinkOnly: 4 # Time limit when the connection is closed after the uplink is closed, Second
   BufferSize: 64 # The internal cache size of each connection, kB
 Nodes:
-  - PanelType: "SSpanel" # Panel type: SSpanel, NewV2board, PMpanel, Proxypanel, V2RaySocks, GoV2Panel
+  - PanelType: "SSpanel" # Panel type: SSpanel, NewV2board, PMpanel, Proxypanel, V2RaySocks, GoV2Panel，AirGo
```
Comment (thank243) [inline release/config/config.yml.example:16]: The comment should use `, AirGo`

File: api/airgo/model.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+type NodeInfoResponse struct {
+	ID             int64  `json:"id"`
+	NodeSpeedlimit int64  `json:"node_speedlimit"`
+	TrafficRate    int64  `json:"traffic_rate"`
+	NodeType       string `json:"node_type"`
+	Remarks        string `json:"remarks"`
+	Address        string `json:"address"`
+	Port           int64  `json:"port"`
+	Scy            string `json:"scy"`
```
Comment (thank243) [inline api/airgo/model.go:13]: The key shoule have some comment.

Comment (ppoonk) [reply api/airgo/model.go:13]: Why are other panels not needed?

Comment (thank243) [reply api/airgo/model.go:13]: I don't know what the `scy` 's meaning. Its not a standard word.

**Pull request closed without merging**